pkg/parsers/operatingsystem/operatingsystem_windows.go

```diff
@@ -62,3 +62,9 @@ func GetOperatingSystemVersion() (string, error) {
 func IsContainerized() (bool, error) {
 	return false, nil
 }
+
+// IsWindowsClient returns true if the SKU is client. It returns false on
+// Windows server.
+func IsWindowsClient() bool {
+	return windows.RtlGetVersion().ProductType == verNTWorkstation
```

Comment:
missing ver != nil check (which, I assume, was here for a reason)

Reply:
It was there originally because of the (incorrect) assumption it could return `nil`, but it always returns a value; https://github.com/moby/moby/blob/0d04359ec2a190c0eecc0cb3c43eae95eb57de60/vendor/golang.org/x/sys/windows/syscall_windows.go#L1489-L1500